packages/expo-updates/e2e/__tests__/fixtures/project_files/e2e/tests/utils/update.ts

```diff
@@ -72,7 +73,7 @@ async function copyAssetToStaticFolder(sourcePath, filename) {
 /**
  * Common method used in all the tests to create valid update manifests
  */
-function getUpdateManifestForBundleFilename(date, hash, key, bundleFilename, assets) {
+function getUpdateManifestForBundleFilename(date: { toISOString: () => any; }, hash: any, key: any, bundleFilename: any, assets: any) {
```

Comment:
```suggestion
function getUpdateManifestForBundleFilename(date: { toISOString: () => any; }, hash: any, key: any, bundleFilename: any, assets: any) {
```

It would be nice to define more precise types for the parameters, we should try to avoid using `any` is possible.

Reply:
Done